test/framework/easyconfig.py

```diff
@@ -4052,20 +4052,26 @@ def test_cuda_compute_capabilities(self):
             "toolchain = SYSTEM",
             "cuda_compute_capabilities = ['5.1', '7.0', '7.1']",
             "installopts = '%(cuda_compute_capabilities)s'",
+            "preinstallopts = '%(cuda_cc_space_sep)s'",
+            "postinstallcmds = '%(cuda_cc_semicolon_sep)s'",
```

Comment:
please use `prebuildopts` here instead, `postinstallcmds` doesn't make any sense at all (that should be a list of strings representing commands, not a single string value)

Reply:
Done